### PR TITLE
More migration to new hooks.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: explore/src/main/scala/explore/targeteditor/AladinCell.scala [hunk 1]
```diff
@@ -76,29 +76,28 @@ object AladinCell extends ModelOptics {
       // flag to trigger centering. This is a bit brute force but
       // avoids us needing a ref to a Fn component
       .useStateViewWithReuse(false)
-      // GuideStar candidates (loading, list)
-      .useState((true, List.empty[GuideStarCandidate]))
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/AladinCell.scala:80]: Why is this gone?it seems we'd need to store the state somewhere?

Comment (rpiaggio) [reply explore/src/main/scala/explore/targeteditor/AladinCell.scala:80]: The state is within the `useStreamBy` hook.

File: explore/src/main/scala/explore/config/GeneratedSequenceViewer.scala [hunk 2]
```diff
@@ -28,16 +28,12 @@ object GeneratedSequenceViewer {
   val component =
     ScalaFnComponent
       .withHooks[Props]
-      .useStreamOnMountBy { props =>
+      .useEffectResultOnMountBy { props =>
```
Comment (cquiroz) [inline explore/src/main/scala/explore/config/GeneratedSequenceViewer.scala:31]: I wonder if we'd need a hook that would take deps and be reactivated if the deps change

File: explore/src/main/scala/explore/targeteditor/AladinCell.scala [hunk 3]
```diff
@@ -76,28 +75,24 @@ object AladinCell extends ModelOptics {
...
-            decodeFromTransferable[IO, CatalogResults](m)(gsl =>
-              gs.setState((false, gsl.candidates)).to[IO]
+      // Listen on web worker for messages with catalog candidates
+      .useStreamBy((props, _, _, _) => (props.target.get, props.obsConf))((props, _, _, _) =>
+        deps =>
+          props.ctx.worker.stream
+            .map(decodeFromTransferable[CatalogResults])
+            .unNone
+            .map(_.candidates)
+            .merge(
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/AladinCell.scala:85]: Looks longger than the original

**Review by cquiroz — COMMENTED** [review]
Some comments

File: explore/src/main/scala/explore/config/GeneratedSequenceViewer.scala (hunk 2 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/config/GeneratedSequenceViewer.scala:31]: There's a version that does that: `useEffectResultWithDepsBy`

File: explore/src/main/scala/explore/targeteditor/AladinCell.scala (hunk 3 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/targeteditor/AladinCell.scala:85]: Well, it was 20 lines before and now it's 16.

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**